### ISSUE TITLE
--dump clarity

### ISSUE THREAD

**Issue by MattHJensen** [open-source-economics/Tax-Calculator#1639]
The tc --dump option documentation is not clear about whether it dumps calculated variables from current law or the reform. 

Ideally there would be an option to dump both in the same file where the current_law and reform versions of each calculated variable are listed side by side. 

Here is what the docs say now: 

```
--dump           optional flag that causes OUTPUT to contain all INPUT
                   variables (possibly extrapolated to TAXYEAR) and all
                   calculated tax variables, where all the variables are named
                   using their internal Tax-Calculator names. No --dump option
                   implies OUTPUT contains minimal tax output.

```

**Comment by martinholmer** [comment]
@MattHJensen said:
> The tc --dump option documentation is not clear about whether it dumps calculated variables from current law or the reform.

All tc --dump and --sqldb output are for the specified policy, which is current-law is no --reform option is specified and is some other policy if the --reform option is used.

I'll try to clarify the documentation.

**Comment by martinholmer** [comment]
@MattHJensen said:
> Ideally there would be an option to dump both in the same file where the current_law and reform versions of each calculated variable are listed side by side.

You want hundreds of variables for evert filing unit for current-law and for the reform in a single file?
How would that file be organized "side by side"?  It is three dimensional.

Your better bet is to do two tc runs: one current-law and the other the reform.
Then compare the two dump files with your favorite graphical diff utility, which will show the files "side by side" and highlight the numbers that differ between the two files.  This is far more efficient that having the numbers all in one file.  The highlighting of the numbers that are different makes it very easy to see the differences in the output from the two runs.

I use this approach all the time.

**Comment by MattHJensen** [comment]
> Your better bet is to do two tc runs: one current-law and the other the reform.
> Then compare the two dump files with your favorite graphical diff utility, which will show the files "side by side" and highlight the numbers that differ between the two files. This is far more efficient that having the numbers all in one file. The highlighting of the numbers that are different makes it very easy to see the differences in the output from the two runs.
> 

Thanks, that's a good tip and I will follow it. I'd guess most users are not familiar with graphical diff utilities. Would you be willing to add a tip about this workflow to the docs?

**Comment by martinholmer** [comment]
@MattHJensen asked:
> Would you be willing to add a tip about this workflow to the docs?

Yes, I was planning to add a discussion of this approach in the [user documentation](http://open-source-economics.github.io/Tax-Calculator/).

**Comment by martinholmer** [comment]
Pull request #1661 resolves issue #1639.
